[2022-01-19T16:43:31.949357+00:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2022-01-19T16:43:31.970896+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-19T16:43:31.970911+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T16:43:31.970922+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T16:43:36.208657+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 2002): An exception occurred in the driver: SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:103)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 2002): SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 2002): SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-01-19T16:46:16.423723+00:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2022-01-19T16:46:16.439517+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-19T16:46:16.439526+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T16:46:16.439531+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T16:46:18.593357+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 2002): An exception occurred in the driver: SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:103)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 2002): SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 2002): SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-01-19T17:00:44.109252+00:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2022-01-19T17:00:44.657881+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-19T17:00:44.657893+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T17:00:44.657900+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T17:00:49.230679+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 2002): An exception occurred in the driver: SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:103)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 2002): SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 2002): SQLSTATE[HY000] [2002] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-01-19T17:05:01.374219+00:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2022-01-19T17:05:01.944463+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-19T17:05:01.944474+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T17:05:01.944480+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-19T17:05:05.051544+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Base 'picshare_test' inconnue" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Base 'picshare_test' inconnue at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:103)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Base 'picshare_test' inconnue at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'picshare_test' inconnue at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-01-31T13:52:43.453841+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\SecurityController::login"},"request_uri":"http://localhost/connexion","method":"POST"} []
[2022-01-31T13:52:43.459411+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-31T13:52:43.459421+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-31T13:52:43.833293+00:00] doctrine.DEBUG: SELECT t0.id AS id_1, t0.email AS email_2, t0.roles AS roles_3, t0.password AS password_4, t0.firstname AS firstname_5, t0.lastname AS lastname_6, t0.created_at AS created_at_7, t0.updated_at AS updated_at_8, t0.token AS token_9, t0.verified AS verified_10, t0.pw_token AS pw_token_11 FROM `user` t0 WHERE t0.email = ? LIMIT 1 ["guillaumeburgnies@hotmail.fr"] []
[2022-01-31T13:52:43.885383+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 69 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 1054): An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:69)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1054): SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Statement.php:95)"} []
[2022-01-31T13:54:09.119028+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\SecurityController::login"},"request_uri":"http://localhost/connexion","method":"POST"} []
[2022-01-31T13:54:09.123722+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-31T13:54:09.123730+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-31T13:54:09.181634+00:00] doctrine.DEBUG: SELECT t0.id AS id_1, t0.email AS email_2, t0.roles AS roles_3, t0.password AS password_4, t0.firstname AS firstname_5, t0.lastname AS lastname_6, t0.created_at AS created_at_7, t0.updated_at AS updated_at_8, t0.token AS token_9, t0.verified AS verified_10, t0.pw_token AS pw_token_11 FROM `user` t0 WHERE t0.email = ? LIMIT 1 ["guillaumeburgnies@hotmail.fr"] []
[2022-01-31T13:54:09.192589+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 69 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 1054): An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:69)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1054): SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Statement.php:95)"} []
[2022-01-31T13:56:07.171648+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\SecurityController::login"},"request_uri":"http://localhost/connexion","method":"POST"} []
[2022-01-31T13:56:07.177530+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2022-01-31T13:56:07.177538+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2022-01-31T13:56:07.260940+00:00] doctrine.DEBUG: SELECT t0.id AS id_1, t0.email AS email_2, t0.roles AS roles_3, t0.password AS password_4, t0.firstname AS firstname_5, t0.lastname AS lastname_6, t0.created_at AS created_at_7, t0.updated_at AS updated_at_8, t0.token AS token_9, t0.verified AS verified_10, t0.pw_token AS pw_token_11 FROM `user` t0 WHERE t0.email = ? LIMIT 1 ["guillaumeburgnies@hotmail.fr"] []
[2022-01-31T13:56:07.273059+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list" at C:\Users\guill\Desktop\picshare2\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 69 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 1054): An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:69)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1054): SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Champ 't0.token' inconnu dans field list at C:\\Users\\guill\\Desktop\\picshare2\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Statement.php:95)"} []
